dev_guide/deployments.adoc

```diff
@@ -247,7 +247,7 @@ triggers:
 === Image Change Trigger
 
 The ImageChange trigger results in a new deployment whenever the value of an
-image stream tag changes.
+image stream tag changes either by a build or because it was imported.
```

Comment:
yeah, without the CCT (but with an ICT in automatic mode) the DC will be updated w/ the new image information (at a minimum this will happen when a deployment is explicilty requested), but no deployment will be triggered automatically. I believe.